app/javascript/packs/application.tsx

```diff
@@ -25,7 +25,7 @@ import * as Student from '../components/student'
 // Add all react components here.
 // Each should map 1-1 to a component in app/helpers/components
 initReact({
-  "example-submissions-summary-table": (data: any) => (
+  'example-submissions-summary-table': (data: any) => (
```

Comment:
At some point we could consider adding a type instead of `any`. We can do that later though.